src/inputs/BasicInputs/BasicInputWrapper/BasicInputWrapper.js

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Cosmo Tech.
+// Lisenced under the MIT lisence.
+
+import { Grid, Typography, makeStyles } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import React from 'react';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import { FadingTooltip } from '../../../misc';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  label: {
+    display: 'flex',
+  },
+  iconTooltip: {
+    marginLeft: '2px',
+    color: theme.palette.text.secondary,
+  },
+}));
+
+export const BasicInputWrapper = ({
+  children,
+  label,
+  tooltipText,
+  containerProps,
+  labelProps,
+  iconTooltipStyle,
+  ...otherProps
+}) => {
+  const classes = useStyles();
+
+  return (
+    <Grid container alignContent="flex-start" className={classes.root} {...containerProps} {...otherProps}>
+      <Grid item className={classes.label}>
+        <Typography {...labelProps}>{label}</Typography>
+        {tooltipText && (
+          <FadingTooltip title={tooltipText}>
+            <InfoOutlinedIcon className={classes.iconTooltip} style={iconTooltipStyle} fontSize="inherit" />
+          </FadingTooltip>
+        )}
+      </Grid>
+      <Grid item>{children}</Grid>
+    </Grid>
+  );
+};
+
+BasicInputWrapper.propTypes = {
+  /**
+   * Component children
+   */
+  children: PropTypes.node,
```

Comment:
Isn't this prop `required` ?

Reply:
Finally, there is a case where the wrapper has no children : when it is directly passed as props to KeyboarDatePicker in BasicDateInput.